oep-template.rst

```diff
@@ -85,32 +77,20 @@ An OEP that introduces backward incompatibilities must describe the
 incompatibilities, with their severity and an explanation of how you propose to
 address these incompatibilities.
 
-OEP submissions that do not consider backward compatibility issues are unlikely
-to be accepted.
-
-
 Reference Implementation
 ========================
 
 The reference implementation must be completed before any OEP is given "Final"
-status, but it need not be completed before the OEP is accepted. While there is
+status, but it need not be completed before the OEP is "Accepted". While there is
 merit to the approach of reaching consensus on the specification and rationale
 before writing code, the principle of "rough consensus and running code" is
-still useful when it comes to resolving many discussions of API details.
-
-The final implementation must include test code and documentation, following
-the `Open edX Contribution Guidelines`_.
-
-.. _Open edX Contribution Guidelines: http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/process/index.html
-
+still useful when it comes to resolving many discussions.
 
 Rejected Alternatives
 =====================
 
 This statement describes any alternative designs or implementations that were
-considered and rejected, and why they were not implemented. It should also link
-to the original source of that discussion.
```

Comment:
It seems like a link to other discussions is still valuable to include, if it's available.

Reply:
See [above comment](https://github.com/edx/open-edx-proposals/pull/53/files#r166806793).

Reply:
I appreciate the effort to reduce barriers to writing.  I think we should leave something here about links because it would be easy to forget to include them.  How about, "Links to the original discussion will be helpful, if you have them."

Reply:
Sure.  But in which section of the document should discussion links go?

Reply:
Given my recent experience with [OEP-17](https://github.com/edx/open-edx-proposals/pull/54), the **Rationale** section seems to be a good fit for links to external discussions.  I'll update this PR to remove it from Rejected Alternatives, but keep it in Rationale.

Reply:
done.